apis/apis.json

```diff
@@ -5156,7 +5156,7 @@
       "id": "Google.Cloud.Translation.V2",
       "productName": "Google Cloud Translation",
       "productUrl": "https://cloud.google.com/translate/",
-      "version": "3.2.0",
+      "version": "3.3.0",
       "type": "rest",
       "description": "Recommended Google client library to access the Translate v2 API. It wraps the Google.Apis.Translate.v2 client library, making common operations simpler in client code. The Translate API translates text from one language to another.",
       "dependencies": {
```

Comment:
I think we should update the Discovery dependency to Google.Apis.Translate.v2 1.67.0.875

Reply:
Right, will do that and BigQuery in a separate PR, then do the release again.